reusable/images_to_numpy.py

```diff
@@ -1,34 +1,76 @@
 import os
 import cv2
+import argparse
 import numpy as np
 from tqdm import tqdm
 from PIL import Image
 import matplotlib.pyplot as plt
 
-# Assuming Folder Structure
 """
+Assuming Dataset Folder Structure.
```

Comment:
Do like this: 

"""
Script to convert a folder of images into numpy format.

Folder structure for images:
Dataset:
    - Images
        - image_01
        - image_02
        - image_03
"""